components/TxForm.tsx

```diff
@@ -64,6 +65,13 @@ export const TxForm = ({ recipientAddress }: { recipientAddress?: string }) => {
   const needsApproval =
     selectedToken?.address !== ethAddress &&
     ethers.utils.parseUnits(tokenAmount || '0', selectedToken?.decimals).gt(tokenAllowance);
+  const onCopy = () => {
+    navigator.clipboard.writeText(`${window.location.host}/send?address=${recipient}`);
```

Comment:
Probably should error/not copy if the address is empty or invalid? Feel free to pull this out into a separate issue

Reply:
tracking in #87